gamemodes/terrortown/gamemode/shared/lang/de.lua

```diff
@@ -1359,8 +1359,22 @@ L.shopeditor_desc_random_shops = [[Zufalls-Shops geben jedem Spieler eine zufäl
 Team-Shops zwingen alle Spieler aus einem Team zu ein und demselben Shopinhalt.
 Auswürfeln erlaubt es ein gegebenes Set für Credits durch ein neues Zufallsset auszutauschen.]]
 L.shopeditor_name_random_shop_items = "Anzahl zufälliger Equipments"
-L.shopeditor_desc_random_shop_items = "Dies enthält Equipments, welche mit .noRandom (immer enthalten) markiert sind. Also wähle die Anzahl hoch genug, damit nicht nur solches Equipment im Shop landet."
+L.shopeditor_desc_random_shop_items = "Dies enthält Equipments, welche mit \"Nicht zufällig\" (immer enthalten) markiert sind. Also wähle die Anzahl hoch genug, damit nicht nur solches Equipment im Shop landet."
 L.shopeditor_name_random_team_shops = "Aktiviere Team-Shops"
 L.shopeditor_name_random_shop_reroll = "Aktiviere Möglichkeit Shop neu auszuwürfeln"
 L.shopeditor_name_random_shop_reroll_cost = "Kosten pro Auswürfeln"
 L.shopeditor_name_random_shop_reroll_per_buy = "Automatisches Auswürfeln nach jedem Kauf"
+
+-- 2021-06-04
+L.header_equipment_setup = "Equipment Einstellungen"
+L.header_equipment_value_setup = "Balance Einstellungen"
+
+L.itemeditor_name_not_buyable = "Nicht kaufbar"
+L.itemeditor_desc_not_buyable = [[Nicht zufälliges Equipment wird im Zufalls-Shop immer enthalten sein.
+Equipments können auf einen Kauf limitiert werden. Entweder Global, im Team oder nur für einen Spieler.]]
+L.itemeditor_name_not_random = "Nicht zufällig"
```

Comment:
I think "nicht zufällig" isn't really descriptive. Equipment marked with `.noRandom` is always included in the shop, even if a random shop is used. Maybe you should make this more clear?

Reply:
Thinking about it a bit more, keep it like you did. Once you finished your todos here, I'll go over the menu and add a few explanatory things. It's easier to do it by myself

Reply:
"Always available"/"Immer verfügbar" would be better. I guess because it had to fit without description and in the old shopeditor .norandom was born.
We could change that now to a better name, but it would destroy every config if we change the NoRandom-name directly.
Instead of course we could just change the translation.

In general I dislike the negation in those variables. Didnt we decide that we wanna do all variables in a non-negated way?

Reply:
yes. It should be done for buyable, I think it is okay for the other variables though. Teamlimited is not really negated for example.

But you can keep it that way and I'll do it quick